capstone/src/main/webapp/dashboard.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="/style.css">
+    <!-- TODO(biancamacias): revist url to add api key -->
+    <!-- <script src="https://maps.googleapis.com/maps/api/js?key="></script> -->
+    <script src="/script.js"></script>
+  </head>
+  <body>
+    <div id="header">
+      <h1>UniViz</h1>
```

Comment:
Not particularly important now and doesn't require any immediate changes, but I believe Michaela used `<h2>` for the `index.html` file. When we start applying CSS, headings should be as consistent as is reasonable.

Reply:
hmmm. ok. we can talk about this in a capstone meeting.